README.md

```diff
@@ -388,6 +409,55 @@ Only expose foreign keys for the comment model findById method. eg. `GET /api/co
 - Type: `boolean|array`
 - Default: `false`
 
+## Custom remote
+
+### `jsonapi` remote options
+Sometime you need to control if custom remote should be serialized or not.
+**By default a custom remote will NOT be handled by JSONApi.**
+
+To enabled/disable JSONApi for specific remotes, you have multiple solutions:
+
+1. Use `jsonapi` remote option
+2. Use `exlude` on component options (see above)
+3. Use `include` on component options (see above)
+4. Use `handleCustomRemote` on component options (see above)
+
+**This option has precedence** and it forces the remote to BE or to NOT BE deserialized/serialized by JSONApi.
+
+#### examples
+```js
+Post.remoteMethod('greet', {
+  jsonapi: true
+  returns: { root: true }
+})
+```
+Ensure the response of `Post.greet` will follow the JSONApi format.
+
+```js
+Post.remoteMethod('greet', {
+  jsonapi: false
+  returns: { arg: 'greeting', type: 'string' }
+})
+```
+Ensure the response of `Post.greet` will not follow the JSONApi format.
+
+#### Note
+You should always pass `root: true` to the `returns` object when you use JSONApi, especialy when you expect to respond with an array.
+
+### Override serialization type
+You may want for a custom method of a given model to return instance(s) from another model. When you do so, you need to enforce the return `type` of this other model in the definition of the remote method.
```

Comment:
When `loopback-component-jsonapi` serializes a custom remote method, by default it will assume that the data being serialized is of the same type as the model the custom remote method is being defined on. Eg. For a remote method on a `Comment` model, it will be assumed that the data being returned from the remote method will be a comment or an array of comments. When this is not the case, you will need to set the type property in the `returns` object in the remote method definition.